scripts/prepare_sector_network.py

```diff
@@ -1111,13 +1111,8 @@ def add_generation(n, costs, existing_capacities=0, existing_efficiencies=None):
 
     nodes = pop_layout.index
 
-<<<<<<< HEAD
-    fallback = {"OCGT": "gas"}
-    conventionals = options.get("conventional_generation", fallback)
-=======
     conventionals = options["conventional_generation"]
```

Comment:
Looks to me like HEAD here is better than upstream

```suggestion
        fallback = {"OCGT": "gas"}
        conventionals = options.get("conventional_generation", fallback)
```

Reply:
Do we have specific code on this ? Upstream code : https://github.com/PyPSA/pypsa-eur/blob/master/scripts/prepare_sector_network.py#L1114

Reply:
I guess we previously improved it on our fork. I'd like to keep our changes and then contribute them to upstream but I can't see all the changes we have here now due to the diffusion...